lib/ramfs/ramfs.h

```diff
@@ -48,17 +48,40 @@ struct ramfs_node {
 	size_t rn_size;    /* file size */
 	char *rn_buf;    /* buffer to the file data */
 	size_t rn_bufsize;    /* allocated buffer size */
-	struct timespec rn_ctime;
-	struct timespec rn_atime;
-	struct timespec rn_mtime;
-	int rn_mode;
-	bool rn_owns_buf;
+	struct timespec rn_ctime;  /* last change time */
+	struct timespec rn_atime; /* last access time */
+	struct timespec rn_mtime; /* last modification time */
+	int rn_mode; /* node access mode */
+	bool rn_owns_buf; /* if the rn_buf was allocated in this ramfs_node */
 };
 
+/**
+ * Allocates ramfs node
+ *
+ * @param name
+ *   the name of the file/directory
+ * @param type
+ *   the type (file or directory)
+ * @return
+ *   the ramfs node
```

Comment:
```suggestion
 *   the new ramfs_node structure
```
I would refer to the structure with it's exact name (`ramfs_node`) so Doxygen will link it to the `ramfs_node` structure page.